src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceCallbackHandler.java

```diff
@@ -96,10 +98,16 @@ protected Map<String, Callback> callbacks() {
 
     private CallbackResponse populateClaimantResponseScenarioFlag(CallbackParams callbackParams) {
         var caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder updatedData = caseData.toBuilder();
 
-        CaseData.CaseDataBuilder updatedData = caseData.toBuilder()
-            .claimantResponseScenarioFlag(getMultiPartyScenario(caseData))
-            .caseAccessCategory(CaseCategory.UNSPEC_CLAIM);
+        if (V_1.equals(callbackParams.getVersion())
+            && featureToggleService.isAccessProfilesEnabled()) {
+            updatedData.claimantResponseScenarioFlag(getMultiPartyScenario(caseData))
+                .caseAccessCategory(CaseCategory.UNSPEC_CLAIM);
+        } else {
+            updatedData.claimantResponseScenarioFlag(getMultiPartyScenario(caseData))
+                .superClaimType(UNSPEC_CLAIM);
```

Comment:
you might need to fix here as well https://github.com/hmcts/civil-service/pull/1646/files#diff-5348adb1f57c5395d552407e44110cec0b0f3a6b4f71d3a689daede01ad9ec59R255